Frontend/app/problem/display/displayProblem.html

```diff
@@ -24,9 +24,15 @@
         </ul>
       </span>
     </div>
-    <p class="lead" *ngIf="problem">
-      Equation I'm trying to solve: <em>{{problem}}</em>
-    </p>
+    <div *ngIf="problem">
+      <p class="lead">
+        <img id="mentii-avatar" src="../../../favicon/mentii-avatar2.png" alt="Mentii Logo">
```

Comment:
Can you put this in a different folder? The favicon folder should be reserved for icon that are used as the icon displayed by the browsers for the website.